internal/databases/mongo/archive/layout.go

```diff
@@ -98,40 +97,35 @@ func LastKnownInBackupTS(backups []models.Backup) (models.Timestamp, error) {
 	return minTS, nil
 }
 
-// SplitPurgingBackups partitions backups to delete and retain
-func SplitPurgingBackups(backups []models.Backup,
-	retainCount *int,
-	retainAfter *time.Time) (purge, retain []models.Backup, err error) {
-	sort.Slice(backups, func(i, j int) bool {
-		return backups[i].StartLocalTime.After(backups[j].StartLocalTime)
-	})
-
-	var backup models.Backup
-	retainedCount := 0
+func GetMongoBackups(backups []models.Backup, purgeBackups, retainBackups map[string]bool) (purge, retain []models.Backup) {
```

Comment:
This function is different from the 'GetRedisBackups'. It makes sense to bring them to the same form